Add Mars's first blog post

MarsBarLee opened the pull request:
Added a markdown file of my blog post, and created a folder and added accompanying images.

Let me know if there's anything else to be done.

rgommers commented on posts/2020/02/my-unexpected-dive-into-open-source-python.md:
```diff
@@ -0,0 +1,61 @@
+
+Before ending this post, I would like to recap a few things.
+
+**There were some initial barriers that prevented me from joining the open-source space.** Without knowing people involved in the community, such as my friend volunteering there, I wouldn't have known about PyData. Without the financial assistance of the Diversity Scholar program, I wouldn't have been able to afford a ticket as an unemployed graduate. A lack of technical knowledge itself isn't pr ...
```
minor textual: "other capable" --> "otherwise capable" ?

MarsBarLee replied:
I think the difference is
"other capable"- that there's more people out there who could join
"otherwise capable"- when these barriers are put down, these capable people could join

Now I'm not sure myself which one to use now since the difference is minor...

MarsBarLee replied:
I decided to stick with 'other'

rgommers commented on posts/2020/02/my-unexpected-dive-into-open-source-python.md:
```diff
@@ -0,0 +1,61 @@
+.. type: text
+-->
+![Header Image](/images/codesprite-comics/My-Unexpected-Dive-Into-Open-Source-Python-Illustration.png)
+*Header illustration by author, Mars Lee*
```
This isn't showing up. Maybe just need a blank line below `-->` (just guessing)?

MarsBarLee replied:
Added the blank space now

MarsBarLee commented:
> Could you try and compress the PyData NYC banner? It's almost 2 MB, which is a little large.
OK! Cmpressed into JPG so its now its 275 KB.

rgommers commented on posts/2020/02/my-unexpected-dive-into-open-source-python.md:
```diff
@@ -0,0 +1,62 @@
+
+At that point, it's been a few months since I graduated from a coding bootcamp. I was feeling down in the job-search funk. I hadn't even done much in Python, since my focus was in Javascript.
+
+![PyData NYC Banner](/images/codesprite-comics/PyData-NYC-Banner.png)
```
this should be `.jpg` not `.png`

rgommers commented:
Thanks @MarsBarLee. The header image shows now (and looks great!), but the PyData banner went missing. You can check the CI outputs like so:

<img width="697" alt="image" src="https://user-images.githubusercontent.com/98330/74807320-ffe82080-529c-11ea-841c-82240121d5fb.png">

Click on "Details" there. That will take you to CircleCI, where you should see an "Artifacts" tab (if you don't, you may have to log in with your GitHub credentials, button in right-top corner):

<img width="697" alt="image" src="https://user-images.githubusercontent.com/98330/74807382-2d34ce80-529d-11ea-89d2-b3ecb2486ff1.png">

Then clicking on the .html file of your post will give you the rendered version as it will appear when we merge this PR.

MarsBarLee commented:
Updated the path for the PyData banner and squashed commits. Thanks @rgommers for the steps on using CircleCI! I haven't used it before and I caught an error. The 'author' section wasn't added by default when I created the markdown file. I corrected it and added myself as an author.

CircleCI also helped me see some errors like the missing blank line after the metadata of the blog's markdown file in my code editor's markdown preview.

MarsBarLee commented:
@rgommers Yes, this Friday 21 Feb sounds good!

rgommers commented:
In it goes, thanks @MarsBarLee!
